packages/discovery-provider/ddl/pg_migrate.sh

```diff
@@ -34,6 +34,47 @@ create_migrations_table() {
     fi
 }
 
+copy_temp_data() {
+    local file=$1
+    local data_dir="./data/$(basename "$file" .sql)"
+    local created_tables=()
+
+    if [[ -d $data_dir ]]; then
+        csv_files=$(ls "$data_dir"/*.csv 2>/dev/null)
+        for csv_file in $csv_files; do
+            table_name="temp_$(basename "$csv_file" .csv)"
+            created_tables+=("$table_name")
+
+            # Read CSV header to infer column names and types
```

Comment:
I didn't see any indication of column types in the CSV...

Reply:
removing this comment 😓